tests/extensions/provisioning/scaleUp.go

```diff
@@ -0,0 +1,20 @@
+package provisioning
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/rancher/tfp-automation/config"
+	set "github.com/rancher/tfp-automation/framework/set"
+	"github.com/stretchr/testify/require"
+)
+
+// ScaleUp is a function that will run terraform apply and scale up the provisioned cluster.
+func ScaleUp(t *testing.T, clusterName string, terraformOptions *terraform.Options, clusterConfig *config.TerratestConfig) {
```

Comment:
I still don't understand / remember why this needs to be separate from ScaleDown. Couldn't you just take the Nodepools as a var instead of the entire config, and make a generic `scale` function?

Reply:
We had that before, but we elected to go this route because when testing scaling, it plays nicer with Terratest having them separate instead of trying to consolidate.

Reply:
Resolved offline in discussion